src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioTransformationServiceHandleKedaInvocationForMediaRequestsTest.java

```diff
@@ -152,7 +154,7 @@ public void handleKedaInvocationForMediaRequestsShouldFailAndUpdateRequestStatus
 
         var notificationEntity = scheduledNotifications.get(0);
         assertEquals(NotificationApi.NotificationTemplate.ERROR_PROCESSING_AUDIO.toString(), notificationEntity.getEventId());
-        assertNotNull(notificationEntity.getTemplateValues());
+
```

Comment:
Add an assert here to cover `notificationEntity.templateValues` are as expected, rather than just not null.